### ISSUE TITLE
qobj_to_circuits does not work with conditionals

### ISSUE THREAD

**Issue by jaygambetta** [Qiskit/qiskit-terra#1630]
### What is the expected behavior?

Follow on after #1629 . The qobj_to_circuits should work with conditional operations. It never did but i noticed in #1629 that it fails. This we need to make a test for as well.

**Comment by kdk** [comment]
Closing in favor of #1952.
